dolphinscheduler-server/src/main/java/org/apache/dolphinscheduler/server/master/runner/MasterTaskExecThread.java

```diff
@@ -74,6 +74,9 @@ public TaskInstance getTaskInstance(){
     public Boolean submitWaitComplete() {
         Boolean result = false;
         this.taskInstance = submit();
+        if(this.taskInstance == null){
+            return result;
```

Comment:
should this return be checked whether the process instance is can be continue normally?
i am not sure about this.

Reply:
@qiaozhanwei  how about your idea

Reply:
Should we send alert here if submit failed?

Reply:
1，Uncertain return, it is recommended to debug
2，submit failed should be logs and mail

Reply:
yes, I debug for submit fail test, process will end and be a failure state.
it is satisfied the result.